<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>373</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="50,50">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_ServerIP">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Connect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Disconnect">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Timing">
         <property name="text">
          <string>Timing(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="horizontalSlider_Timing">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_Timing"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_Start">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Stop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Plotter" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_Timing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_Timing</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Disconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startTemp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopTemp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Update</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>insertMachines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tcpConnect()</slot>
  <slot>tcpDisconnect()</slot>
  <slot>startTemp()</slot>
  <slot>stopTemp()</slot>
  <slot>showProd()</slot>
  <slot>insertMachines()</slot>
 </slots>
</ui>
